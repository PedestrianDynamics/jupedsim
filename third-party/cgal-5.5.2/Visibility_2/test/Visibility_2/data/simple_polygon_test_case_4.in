# Test case 4 for simple polygons. See https://cgal.geometryfactory.com/CGAL/Members/wiki/Visibility/TestCases
8
0 0
4 0
8 3
12 0
12 3
8 6
4 3
0 3
8
0 1
1 2
2 3
3 4
4 5
5 6
6 7
7 0
2 3