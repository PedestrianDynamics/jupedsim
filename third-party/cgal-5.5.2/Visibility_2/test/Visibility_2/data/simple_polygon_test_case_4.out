# Test case 4 for simple polygons. See https://cgal.geometryfactory.com/CGAL/Members/wiki/Visibility/TestCases
6
0 0
4 0
8 3
12 3
4 3
0 3
6
0 1
1 2
2 3
2 4
4 5
5 0